experimental/framework/frs-client/package.json

```diff
@@ -25,6 +25,11 @@
     "eslint": "eslint --format stylish src",
     "eslint:fix": "eslint --format stylish src --fix",
     "lint": "npm run eslint",
+    "start:frs": "webpack-dev-server --define process.env.FLUID_CLIENT='\"frs\"'",
```

Comment:
does this script run the tests?  the npm scripts to run with frs should be in the same format as the tinylicious ones below

Reply:
good catch. I forgot to add in the run mocha script. Fixed with `"test:frs": "npm run start:frs && npm run test:realsvc:run",`